### PR TITLE
Add pulp_container to setup.py

### PR DESCRIPTION
**Pull request by newswangerd**

### CODE REVIEW

**Review by alikins — CHANGES_REQUESTED** [review]
Add the 'why' you mentioned in slack to the commit message to explain the intent of the change.

File: requirements/requirements.standalone.in [hunk 1]
```diff
@@ -1 +0,0 @@
-pulp-container~=2.3.1
```
Comment (alikins) [inline requirements/requirements.standalone.in:1]: If we are getting rid of requirements.standalone.in, can requirements.standalone.txt go as well?

Would need to be removed from dev/standalone/Dockerfile and the Makefile.

Or would/should that be a different pr?

Comment (newswangerd) [reply requirements/requirements.standalone.in:1]: I left it in there just in case we want to add other standalone only dependencies. Not sure if it's the best thing to do or not

Comment (newswangerd) [reply requirements/requirements.standalone.in:1]: I think it would  have to be updated in a bunch of different places

Comment (alikins) [reply requirements/requirements.standalone.in:1]: Seems fine to me to leave it for now. We can remove it later if needed.

**Review by alikins — APPROVED** [review]
LGTM
